src/language-usability.rst

```diff
@@ -16,6 +16,193 @@ Usability of Type Systems
 API Usability
 =============
 
+..
+  Outline
+  * APIs are ubiquitous
+  * Examples
+  * High level goals of API
+  * Difficulty in achieving goals of API
+  * Distinction with language usability
+..
+
+Application Programming Interfaces (APIs) are interfaces that mediate interactions between software components.
```

Comment:
Let's try "Application Programming Interfaces (APIs) are user interfaces that serve a dual role: they are simultaneously the user interfaces that programmers use to interact with software components and formal interfaces that govern interactions between different software components."